cradlepoint/pages/api/editTestPlan.js

```diff
@@ -16,8 +16,9 @@ export default async (req, res) => {
       const result = testPlanSchema.cast(data);
       // Set ID strings to Mongo ObjectId's
       const id = ObjectId(result._id);
+      const summaryBOM = result.summaryBOM.map((mystr) => ObjectId(mystr))
       const query = {_id: id};
-      const testPlan = {...result, _id: id };
+      const testPlan = {...result, summaryBOM, _id: id };
```

Comment:
I think this won't work for how we're doing the summaryBOM now, but it doesn't affect the PR so I'm instead gonna open an issue and we'll deal with it later